011-kafka-canary.md

```diff
@@ -0,0 +1,40 @@
+# Strmixi Canary
+
+Implement a Kafka canary tool which will act as an indicator of whether Kafka clusters are operating correctly. This would be achieved by creating a canary topic and continously producing and consuming events on the topics. 
+
+## Current situation
+
+Currently  Strimzi does not have any canary style feature. It does have readiness checks for Kafka cluster which can be replaced/augmented with the canary.
```

Comment:
What readiness checks do you have in mind? The ones in Kube pods? This cannot replace them because they have completely different semantics. For example they do not focus on Kafka cluster at all. They are pod scoped only. It can of course add additional level of health checking, but it would need to be a separate thing run from somewhere else. 

Or did you meant something else with the readiness checks?

Reply:
Yes, that is exactly my point. This could be used as an additional health check in KafkaRoller. But not as a readiness check in Kafka pods. Hence the need to properly explain what does it mean and avoid any possible confusion.

Reply:
Yeah this was a typo. Fixed now!